### PR TITLE
Javascript module: Restaurant page project

### PR DESCRIPTION
**Pull request by MiguelArgentina**
- The page loads all DOM elements using  Javascript
- All styling is placed in the style.css file
- Webpack was used as required
- Added two scripts: one for developing and one for production

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+- Javascript (ES6)
+
+## Live Demo
+
+Visit the [Restaurant Page](https://flamboyant-ride-be0a79.netlify.app/)
+
+## Prerequisites
+- Internet browser compatible with HTML5, CSS3 and Javascript ES6
+
+## Getting Started
```
Comment (afanimartin) [inline README.md:28]: - Add instruction for installing packages, building using webpack and starting the local server

File: src/tabContact.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import {
+  addElementAsSibling, navBarClassesUpdater,
+} from './addElements';
+
+export default function contactTabContent() {
```
Comment (afanimartin) [inline src/tabContact.js]: - Use ES6 arrow functions syntax

File: src/tabHome.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {
+  addElementAsSibling, navBarClassesUpdater,
+} from './addElements';
+
+export default function ourSpaceTabContent() {
```
Comment (afanimartin) [inline src/tabHome.js]: - Use ES6 syntax for declaring functions

File: src/tabMenu.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import {
+  addElementAsSibling, navBarClassesUpdater,
+} from './addElements';
+
+export default function ourMenuTabContent() {
```
Comment (afanimartin) [inline src/tabMenu.js]: - Use ES6 syntax

File: src/tabReservations.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import {
+  addElementAsSibling, navBarClassesUpdater,
+} from './addElements';
+
+export default function reservationsTabContent() {
```
Comment (afanimartin) [inline src/tabReservations.js]: - Use Es6 syntax

File: package.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve"
+  },
```
Comment (afanimartin) [inline package.json]: - Add the required `dev` command
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this project. You did really well with the design and functionalities of the app.
### Highlights 🌵 🌵 🌵 
- Use of modules
- Required packages added
- Use of tabbed menu to render contents

### Changes 🙅🏿 🙅🏿 🙅🏿 
- Use ES6 syntax to declare functions. The `function` keyword should not appear anywhere in your codebase
- Update readme with commands for installing packages and building app locally


### Martin Afani
#### Slack: whiz25

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow. 💯 
  - App is functionsl and available online. 💯 
  - It has a good design. 💯 

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**